styles/style.css

```diff
@@ -0,0 +1,191 @@
+.container {
+    display: grid;
+    border: solid black 2px;
+    flex-wrap: wrap;
+}
+
```

Comment:
It would be better if you specifified the size or gri of the contained because it is currently causing the right border to be left of the content.